### ISSUE TITLE
Packmode Recipe Change Issue

### ISSUE THREAD

**Issue by tyrobbb94** [SpongePowered/SpongeForge#1964]
I am currently running

- SpongeForge version: spongeforge-1.12.2-2555-8.0.0-BETA-2841
- Forge version: forge-1.12.2-14.23.1.2587
- Java version: 9
- Operating System: Windows 10

- Plugins/Mods: https://rawgit.com/AllTheMods/ATM-3/master/modlist.html

**Issue Description**

Using a pack with alternate pack modes (All The Mods 3 in this case), regardless of what the packmode is set to, the server will not load the recipe changes from the script files. JEI will show the new customized recipe, but no output for the recipe will be available. Instead, the vanilla crafting requirements are needed to make the item. This only occurs when sponge is installed, and works as it should without sponge.

**Comment by ryantheleach** [comment]
The entire Minecraft ecosystem including Forge and Sponge is currently unequipped to run Java 9 in production.

Use Java 8. We will update at our own pace, and using the new Java 9 Module system before everyone is ready is going to end in pain.

Additionally, You are running `spongeforge-1.12.2-2555-8.0.0-BETA-2841`

API 8.0.0-SNAPSHOT versions are listed under the experimental tag on the downloads page.

You should be using builds made for API 7.x.x

**What experimental means:**
Plugins are expected to be releasing builds for API 7.0.0 as that will be compatible with all SpongeForge / SpongeVanilla releases that have a starting 7 in their build number. e.g.

SpongeForge 7.0.1, 7.1.0 7.2.0-SNAPSHOT all will be compatible with plugins that are built against API 7.0.0 and work correctly.

Running 8 puts pressure on plugin authors to update to builds that the Sponge team is actively breaking and experimenting with. There is 0 reason to be running a server on a 8.0.0 snapshot build at this time _unless_ you are a plugin author looking to test your plugins and give feedback on upcoming API changes.

Occasionally people will use the experimental versions to update to the new version of Minecraft early, but that is on them , and the plugin authors who wish to support that.

8.0.0 is currently estimated by the Sponge team to align with our minecraft 1.13 release. So you can kinda see why running a 1.12 server on an API snapshot that is going to change dramatically for 1.13 might be a bad idea?

It's currently far far too early for us to start even offering support for API 8 builds to end users, let alone plugin authors.

Yes we could communicate this more clearly on the downloads page, and we would accept design suggestions that make this more clear, as it's a problem we have had for like, 12 months now.

**TLDR: NO JAVA 9. NO API 8.**

**Comment by tyrobbb94** [comment]
I apologize, I mistyped with that, and meant to put Java Version 8. Thank you i will look into this.

**Comment by ryantheleach** [comment]
Remake your issue if it still exists on a SpongeForge API 7.x.x latest (stable) builds.

**Comment by tyrobbb94** [comment]
Will do.
